views/components/footer.njk

```diff
@@ -0,0 +1,12 @@
+<footer class="d-flex fixed-bottom flex-wrap justify-content-between align-items-center py-3 border-top">
+    <p class="col-md-4 mb-0">© 2024 Kainos</p>
+    
+    <div class="nav-logo-container">
+        <img src="../assets/index/kainos-logo.png" alt="transparent kainos logo for the navbar" class="img-fluid">
+    </div>
+    
+    <ul class="nav col-md-4 justify-content-end gap-4">
+        <li class="nav-item">123 Kainoos Road</li>
```

Comment:
I love this, but please put the proper Kainos address and company name

Reply:
Done

Reply:
[028 9057 1100](https://www.google.com/search?q=kainos&rlz=1C5GCEM_enGB1120GB1120&oq=kainos&gs_lcrp=EgZjaHJvbWUqBwgAEAAYjwIyBwgAEAAYjwIyEAgBEC4YxwEYsQMY0QMYgAQyBggCEEUYPDIGCAMQRRg8MgYIBBBFGDwyBggFEEUYPDIGCAYQRRg8MgYIBxBFGDzSAQgxNjg3ajBqN6gCCLACAQ&sourceid=chrome&ie=UTF-8#)

Reply:
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/ae34f93d-bfa3-4da1-b501-6a710545a452">